go/logictest/runner.go

```diff
@@ -225,6 +234,11 @@ func skipUntilEndOfRecord(scanner *parser.LineScanner, wr *bufio.Writer) {
 	}
 }
 
+type loggingLock struct {
+	mux *sync.Mutex
```

Comment:
I think (not positive) that sync.Mutex has a ready to use zero value. So if you make this not a pointer, this struct won't need any initialization at all.